jaeger_client/tracer.py

```diff
@@ -41,19 +47,24 @@ class Tracer(opentracing.Tracer):
     N.B. metrics has been deprecated, use metrics_factory instead.
     """
     def __init__(
-        self, service_name, reporter, sampler, metrics=None,
-        metrics_factory=None,
-        trace_id_header=constants.TRACE_ID_HEADER,
-        generate_128bit_trace_id=False,
-        baggage_header_prefix=constants.BAGGAGE_HEADER_PREFIX,
-        debug_id_header=constants.DEBUG_ID_HEADER_KEY,
-        one_span_per_rpc=False, extra_codecs=None,
-        tags=None,
-        max_tag_value_length=constants.MAX_TAG_VALUE_LENGTH,
-        max_traceback_length=constants.MAX_TRACEBACK_LENGTH,
-        throttler=None,
-        scope_manager=None,
-    ):
+        self,
+        service_name: str,
+        reporter: Any,
+        sampler: Sampler,
+        metrics: Optional[Any] = None,
+        metrics_factory: Optional[MetricsFactory] = None,
+        trace_id_header: str = constants.TRACE_ID_HEADER,
+        generate_128bit_trace_id: bool = False,
+        baggage_header_prefix: str = constants.BAGGAGE_HEADER_PREFIX,
+        debug_id_header: str = constants.DEBUG_ID_HEADER_KEY,
+        one_span_per_rpc: bool = False,
+        extra_codecs: Optional[dict] = None,
+        tags: Optional[dict] = None,
+        max_tag_value_length: int = constants.MAX_TAG_VALUE_LENGTH,
+        max_traceback_length: int = constants.MAX_TRACEBACK_LENGTH,
+        throttler: Optional[Any] = None,
```

Comment:
need to add Throttler class and have RemoteThrottler extend it

Reply:
So, should I wait till this is done?

Reply:
if you want to add it, go ahead, but I'm ok to merge as is.

Reply:
I see this outside of this PR to be honest 😄

Reply:
#330